parser/worker.go

```diff
@@ -358,5 +358,8 @@ func (w Worker) ExportTxs(txs []*types.Tx) error {
 	totalBlocks := w.db.GetTotalBlocks()
 	logging.DbBlockCount.WithLabelValues("total_blocks_in_db").Set(float64(totalBlocks))
 
+	dbLatestHeight, _ := w.db.GetLastBlockHeight()
```

Comment:
We should handle the error here. He might never know what happens, and we should make sure this doesn't panic if (for any reason) this returns an error